src/Layer/Layer.js

```diff
@@ -56,6 +53,13 @@ class Layer extends THREE.EventDispatcher {
      * the layer doesn't need Source (like debug Layer or procedural layer).
      * @param {number} [config.cacheLifeTime=Infinity] - set life time value in cache.
      * This value is used for [Cache]{@link Cache} expiration mechanism.
+     * @param {(boolean|Object)} [config.addLabelLayer=false] - Used to tell if this layer has
+     * labels to display from its data. For example, it needs to be set to `true`
+     * for a layer with vector tiles. If it's `true` a new `LabelLayer` is added and attached to this `Layer`.
+     * You could pass [LabelLayer]{@link LabelLayer} options in `addLabelLayer` (`addLabelLayer: { performance: true }`).
```

Comment:
I would say "you can also configure it with options described below such as: `addLabelLayer: { performance: true }`" instead, WDYT?

Reply:
yes